diff --git shapely/libgeos.py shapely/libgeos.py
index 424c5ee..e80fcb2 100644
--- shapely/libgeos.py
+++ shapely/libgeos.py
@@ -95,6 +95,7 @@ if not lgeos and geos_config:
     if sys.platform == 'win32':
         # Unlikely, since geos-config is a shell script, but you never know...
         fmts = ['{name}.dll']
+        base = os.path.join(sys.prefix, "Library", "bin")
     elif sys.platform == 'darwin':
         fmts = ['lib{name}.dylib', '{name}.dylib', '{name}.framework/{name}']
     elif os.name == 'posix':
